src/Sylius/Bundle/CoreBundle/Tests/DataFixtures/ORM/api_administrator.yml

```diff
@@ -0,0 +1,23 @@
+Sylius\Bundle\ApiBundle\Model\Client:
+    oauth:
+        randomId: client_id
+        secret: secret
+        allowedGrantTypes: [password, access_token, refresh_token]
+
+Sylius\Bundle\ApiBundle\Model\AccessToken:
+    oauth_token:
+        client: @oauth
+        user: @admin
+        token: access_token
+
+Sylius\Component\Core\Model\User:
+    admin:
+        plainPassword: sylius
+        roles: [ROLE_API]
```

Comment:
According to data fixtures we use [`ROLE_API`](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadApiData.php#L38)